microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -1,103 +1,110 @@
 ---
-title: Remote Debugging for WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
-description: How to remotely debug WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge. 
+title: Remote debugging for WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
+description: How to remotely debug WebView2 WinUI 2 (UWP) apps by using Remote Tools for Microsoft Edge. 
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 ms.date: 02/23/2023
 ---
-# Remote Debugging WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
+# Remote debugging WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
 
-Use `Remote Tools for Microsoft Edge` to enable [Microsoft Edge DevTools](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/landing/) debugging.
+Use `Remote Tools for Microsoft Edge` to enable [Microsoft Edge DevTools](/microsoft-edge/devtools-guide-chromium/landing/) debugging.
 
 
 <!-- ====================================================================== -->
-## Debugging Desktop WebView2 WinUI 2 (UWP) apps
+## Debugging desktop WebView2 WinUI 2 (UWP) apps
 
-These steps can be used to attach `Microsoft Edge DevTools` remotely to a WebView2 WinUI 2 (UWP) app.
-Remote debugging is necessary as a current limitation to WebView2 WinUI 2 (UWP) apps is the built-in `Microsoft Edge DevTools` cannot be launched inside a store signed WebView2 WinUI 2 (UWP) app.
+To use Microsoft Edge DevTools to debug a desktop WebView2 WinUI 2 (UWP) app, use remote debugging.  Remote debugging is necessary because currently, the built-in DevTools in Microsoft Edge cannot be launched inside a store-signed WebView2 WinUI 2 (UWP) app.
 
-1. Install [Edge Canary](https://www.microsoftedgeinsider.com/en-us/download/canary).
+Attach `Microsoft Edge DevTools` remotely to a WebView2 WinUI 2 (UWP) app as follows:
 
-1. In `Edge Canary` navigate to `edge://inspect` and verify the page is supported in your version of the browser.
+1.  Install [Microsoft Edge Canary Channel](https://www.microsoftedgeinsider.com/download/canary).
 
-1. Enable `Developer Mode`:
+1.  In Microsoft Edge Canary, go to to `edge://inspect` and verify that the **Inspect with Developer Tools** page exists in your version of the browser:

```

Comment:
just a general comment, which is probably a style thing based on our different doc sets. We only use the inline code backtick for inline code, not as a general highlighting tool.

Reply:
@dan-wesley fyi FTS on "edge://" in this repo shows such specialty URLs as using backticks